tests/tests_shared.h

```diff
@@ -113,27 +113,32 @@ extern "C" {
 
 # define SIR_CL_PERFDESC      "Only run the performance measurement test"
 # define SIR_CL_ONLYDESC      "Only run the test(s) specified"
-# define SIR_CL_LISTDESC      "Prints a list of available test names for use with '" BOLD("--only") "'"
+# define SIR_CL_LISTDESC      "Prints a list of available test names for use with '" SIR_BOLD("--only") "'"
 # define SIR_CL_LEAVELOGSDESC "Log files are not deleted so that they may be examined"
 # define SIR_CL_WAITDESC      "After running test(s), wait for a keypress before exiting"
 # define SIR_CL_VERSIONDESC   "Prints the version of libsir that the test suite was built with"
 # define SIR_CL_HELPDESC      "Shows this message"
 
 /** Prints `msg` indented with a tab. Used for output during a test's execution. */
-# define TEST_MSG(msg, ...) (void)printf("\t" msg "\n", __VA_ARGS__)
+# define TEST_MSG(msg, ...) (void)printf("\t" msg SIR_EOL, __VA_ARGS__)
 
 /** TEST_MSG but no varargs. Use when `msg` is just a string. */
-# define TEST_MSG_0(msg) (void)printf("\t" msg "\n")
+# define TEST_MSG_0(msg) (void)printf("\t" msg SIR_EOL)
 
 /** Prints `msg` in red to stderr. */
-# define ERROR_MSG(msg, ...) TEST_MSG(RED(msg), __VA_ARGS__)
+# define ERROR_MSG(msg, ...) TEST_MSG(SIR_RED(msg), __VA_ARGS__)
 
 /** ERROR_MSG but no varargs. Use when `msg` is just a string. */
-# define ERROR_MSG_0(msg) TEST_MSG_0(RED(msg))
+# define ERROR_MSG_0(msg) TEST_MSG_0(SIR_RED(msg))
 
 /** Prints `msg` in green if `expr` is true, or in red if false. */
-# define PASSFAIL_MSG(expr, msg, ...) \
```

Comment:
Aren't `SIR_GREEN` and `SIR_RED` both just the input arg when `SIR_NO_TEXT_STYLING`?

Reply:
If you allow it to be always true or false, you get one of those always true and/or always false warnings that you'd have to suppress otherwise.